vagrant/README.md

```diff
@@ -0,0 +1,29 @@
+# Vagrant
+
+Here you will find a `Vagrantfile` to build a development environment for `kvm-vmi`.
+
+# Requirements
+
+- `vagrant`
+- `vagrant-libvirt` plugin
+- `ansible >= 2.2.1.0` (use a virtualenv, install it via pip)
+
+# Setup
+
+At first, make sure to run `vagrant plugin update vagrant-libvirt` to have
+the latest version
+
+- Then, you just have to run `vagrant up --provider=libvirt`
+- Once vagrant is done creating and provisioning the box, **reboot the box**(from libvirt) to
+run the new kernel with the modified KVM modules, and run `vagrant ssh` to get a shell in the VM.
+- Start the test vm with `virsh start nitro_win7x64` (or use `virt-manager` connection)
+- go to `/data/kvm-vmi/nitro` and you can play with **Nitro** !
+
+# Virt-manager
+
+You can use `virt-manager` to view and manage VMs in the vagrant box.
+SSH is open for password authentication.
+Specify the `root` account with the password `vagrant`.
+
+To have the graphical display working, you need to change the `Display Spice/VNC` configuration:
+set `address` to `All interfaces`.
```

Comment:
I could modify `tests/template_domain.xml` to force the display to listen on `0.0.0.0`, but that would be a security issue if someone install the test VM on bare metal, because the `SPICE/VNC` server would be directly accessible from the network.